pilot_experiment/2_explore_data.ipynb

```diff
@@ -0,0 +1,1556 @@
+       "GSM199985.CEL  0.180992  0.318097  0.321321  \n",
+       "GSM199986.CEL  0.718328  0.219357  0.299589  \n",
+       "\n",
+       "[5 rows x 5549 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "shuffled_expression.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>annotation</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PAO1_gene_id</th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>PA0001</th>\n",
+       "      <td>core</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA0002</th>\n",
+       "      <td>core</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA0003</th>\n",
+       "      <td>core</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA0004</th>\n",
+       "      <td>core</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>PA0005</th>\n",
+       "      <td>core</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "             annotation\n",
+       "PAO1_gene_id           \n",
+       "PA0001             core\n",
+       "PA0002             core\n",
+       "PA0003             core\n",
+       "PA0004             core\n",
+       "PA0005             core"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gene_annot.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Group genes by core and accessory annotation\n",
+    "core_gene_ids = list(gene_annot[gene_annot['annotation'] == 'core'].index)\n",
+    "acc_gene_ids = list(gene_annot[gene_annot['annotation'] == 'accessory'].index)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Examine interactions between genes\n",
+    "\n",
+    "First we can examine how coordinated groups of genes are: core-core, core-accessory, accessory-accessory "
```

Comment:
The genes in this dataset were aligned using the PAO1 reference transcriptome. So the core genes are those that are shared/homologous between PAO1 and PA14 and the accessory genes are those that are specific to PAO1 and not in PA14.

The samples I'm using is indeed a correlation of gene expression profiles using PAO1 and PA14 samples. One of the reason for this combined set is because that way the samples that comprised the expression data matched the gene annotations (i.e. The comparison of PAO1 and PA14 genomes were used to annotate genes, and the samples were both PAO1 and PA14). Second, I expect to see a very strong signal to differentiate core and acccessory genes by combining these two. 

I do think it would be very interesting to see what the correlations look like using only PAO1 and only PA14 samples! I expect the difference between core and accessory will be more subtle. And it would be interesting to see how interactions of genes change depending on the dataset - what determines what relationships we can find in the data?

Reply:
Makes sense, thanks! This seems like a good approach for a pilot study.